### PR TITLE
#159250154 Add hover effect on muscles to show exercises

### PR DESCRIPTION
**Pull request by matthewacha**
### What does this PR do?
A user should be able to hover the cursor over a body part and get exercises for that body part

### Description of Task to be completed?
Currently a user can only view excercises corresponding to the muscle links they hover over. However, the user should also be able to view the excercises of the muscle image.

### How should this be manually tested?
Log in to wger
Navigate to `Exercises` dropdown
Click `Muscle overview`
Hover over the muscles on the body parts to see the exercises

### What are the relevant pivotal tracker stories?
[#159250154](https://www.pivotaltracker.com/story/show/159250154)

[Deliver #159250154]

### Screenshots
image

### CODE REVIEW

File: wger/exercises/templates/muscles/overview.html [hunk 1]
```diff
@@ -43,12 +42,19 @@
...
         <h4 style="padding-bottom:0.5em;">{% trans "Muscles" %}</h4>
         <div id="muscle-system"
              class="muscle-background"
              style="background-image: url('/static/images/muscles/muscular_system_front.svg');">
         </div>
+        <object id="muscle-sys"
+        class="muscle-background front" type="image/svg+xml" data='/static/images/muscles/muscular_front_highlight.svg'></object>
+        <object id="muscle-sys"
+        class="muscle-background back" type="image/svg+xml" data='/static/images/muscles/muscular_back_highlight.svg'></object>
+
```
Comment (mutebironald) [inline wger/exercises/templates/muscles/overview.html:55]: @matthewacha .Alot of spacing after line 54. Is it neccessary?

**Comment by JaymesKat** [comment]
@matthewacha please correct spelling of `exercises` in your commit message header as well as in the PR description

**Comment by JaymesKat** [comment]
@matthewacha Tested the feature on my local machine, it is working well

**Comment by mutebironald4** [comment]
@matthewacha . Fix build

**Comment by mutebironald** [comment]
@matthewacha , fix your build

**Review by mutebironald — APPROVED** [review]

**Review by archbishop12 — APPROVED** [review]

**Comment by isheebo** [comment]
As per this screenshot, no exercises are seen when a muscle is hovered over, please fix this. @matthewacha 
screen shot 2018-08-30 at 6 22 49 pm

**Pull request merged**